src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int HUNDRED = 100;

```

Comment:
Which access modifier should you use here? Also, it's a bad name for constant better use MAX_NUMBER_OF_BALL